client/src/app/components/Toolbar/components/UserBar/UserBar.jsx

```diff
@@ -3,8 +3,8 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import classNames from 'classnames';
 
-import { logoutCurrentUser } from './actions';
-import { getCurrentUser } from 'common/selectors';
+import { logoutCurrentUser } from './model/actions';
```

Comment:
only wondering if actions related with user should be place in different places?

Reply:
actions referring to their components should be placed within this component directory - this is how we decided, so I hope it's in a good place 😃